benchmarks/kubectl-mtb/test/benchmarks/require_always_pull_image/require_always_pull_image.go

```diff
@@ -56,6 +66,23 @@ var b = &benchmark.Benchmark{
 	},
 }
 
+func isAlwaysPullImagesAdmissionPluginEnabled(options types.RunOptions) (bool, error) {
```

Comment:
@mac-chaffee - wondering if checking the apiserver args is the best option. 

If the AlwaysPullImages admission control is enabled it will update the pod `imagePullPolicy` and we can simply check the value. Can we change the check to compare the results, if the operation that creates a pod with `ImagePullPolicy: Never` (or IfNotPresent) is allowed?

Besides, the check will only work for clusters where the `kube-apiserver` is accessible as a pod. That seeems typical, but not sure if its guaranteed.

Reply:
You're right, I forget people can run kube-apiserver as anything other than a mirrored pod like how kubeadm does it.

And yes you're right that the `metav1.DryRunAll` means the admission control plugins do take effect, and you can check the returned Pod object to see if any fields were mutated.